scopes/react/component-highlighter/element-highlighter/element-highlighter.tsx

```diff
@@ -19,43 +22,37 @@ export interface ElementHighlighterProps extends React.HTMLAttributes<HTMLDivEle
 
 export { Placement };
 
-export type HighlightTarget = {
-  /** element to show the highlight at */
-  element: HTMLElement;
-  /** components with metadata to show in the label */
-  components?: ComponentMetaHolder[];
-};
-
 export type HighlightClasses = {
   container?: string;
   frame?: string;
   label?: string;
 };
 
 export function ElementHighlighter({
-  target,
+  targetElement,
+  components,
   placement = 'top',
-  watchMotion = true,
+  watchMotion,
```

Comment:
disabling motion watching by default (still enabled for the hover / children / hybrid highlighter)